README.md

```diff
@@ -32,6 +32,18 @@ Tools supporting PanSN should allow the user to change the delimiter.
 
 The prefixing should provide a unique hierachy of sample names and haplotype identifiers for the entire pangenome under analysis.
 
+### haplotype_id
+
+The use of strings for `haplotype_id` allows for clear representation of various assembly types and scenarios. Here are detailed examples based on common labeling practices from Vertebrate Genome Project (VGP) and Earth BioGenome Project (EBP):
```

Comment:
How do we square this table of haplotype name meanings with the paragraph later on about how PanSN is not a metadata format? Because this ends up looking like we're defining a metadata format where you pick the right haplotype name to encode your metadata, and if you start having new metadata to encode (this is a decoy, this is the third copy in a trisomy) you might want to come back here and register a new name for it.

Maybe this whole section needs to be clearly a [MAY](https://www.rfc-editor.org/rfc/rfc2119). If a tool consuming PanSN-named assemblies actually needs to know which assembly came from which parent, it MUST NOT require that this will be encoded as `mat` and `pat` names. It MUST support a way for the user to specify that the egg genome is `1` and the sperm genome is `2`, or even that the egg genome is `pat` and the sperm genome is `mat`.